src/libs/libRocket/Include/Rocket/Core/StringBase.h

```diff
@@ -185,8 +189,18 @@ class StringBase
 	inline const T& operator[](size_type index) const;
 	inline T& operator[](size_type index);
 
+	// Offer c_str() to make the interface somewhat more consistent with std::string.
+	// It eases code migration to std::string in the future should that be needed.
+	// The name also imparts better expectations about what it does better
+	// than CString() which can be confused with things like MFC's class of the same name.
+	inline const T* c_str() const;
```

Comment:
If I have to comply I will, but unless there is more to this I disagree. Using CString in the first place was probably a bad idea IMO as It doesn't convey what c_str() and doesn't help generic code and it doesn't serve a useful purpose to propagate the situation. Plus it's longer and defeats muscle memory for regular string. It's is also an inline function that has no cost to add now so why wait.